jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ResourceHandler.java

```diff
@@ -111,7 +111,7 @@ public Request.Processor handle(Request request) throws Exception
             return super.handle(request);
         }
 
-        HttpContent content = _resourceService.getContent(request.getPathInContext(), request);
+        HttpContent content = _resourceService.getContent(Request.getPathInContext(request), request);
```

Comment:
Store `Request.getPathInContext(request)` in a local variable to avoid calling it multiple times.